### ISSUE TITLE
CRLF newlines problems

### ISSUE THREAD

**Issue by LaBetonneuse** [piranha/gostatic#26]
Hello, in the last few days I've started to work on a project using gostatic and I can't seem to get it to work as (I) expected. In the result I get, the first page is fine but it seems like the second one is processed but its properties aren't accessible, i.e. the title is empty.
There must be something I'm doing wrong but I can't figure out what and it's frustrating.
Also, I don't know why but the rules in the config file made the files in the static folder to be consiedered as pages O_o. That made me think that maybe a "copy" processor would be welcome, to copy the contents of a folder instead of processing it.
I'm using a markdown for the page that should be processed "as is" and a GoTemplate for the one that's going to need more processing once it get it to render correctly.

My config file looks like this (it has to be UTF-8):
```
TEMPLATES = site.tmpl
SOURCE = src
OUTPUT = site
TITLE = La bétonneuse
AUTHOR = Guillaume Veys - La bétonneuse

*.md:
	config
	ext .html
	markdown
	template page

*.html:
	config
	ext .html
	inner-template
	template page
```

And in the src folder contains index.md:
```
title: Accueil
description: Page d'accueil
----
<h1>Bienvenue</h1>
<p>Bienvenue chez moi !</p>
<p>Texte</p>
```

and logiciels.html:
```
title: Logiciels
description: Mes logiciels
----
<p>Voici la liste des logiciels que je vous propose {{ .Title }} :</p>
<p>
	<ul class="logiciels">
		<li>Logiciel 1</li>
		<li>Logiciel 2</li>
	</ul>
</p>
```

site.tmpl looks like this:
```
{{ define "header" }}<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<?xml version="1.0" encoding="iso-8859-1" ?>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="fr" lang="fr">
<head>
	<meta charset="utf-8">
	<meta name="author" content="{{ html .Site.Other.Author }}">
	<title>{{ if .Title }}{{ html .Title }} - {{ end }}{{ html .Site.Other.Title }}</title>
	<link rel="stylesheet" type="text/css" href="{{ .Rel "resources/style/style.css" }}">
	<link href="{{ .Rel "favicon.ico" }}" type="image/x-icon" rel="shortcut icon" />
</head>
<body>
	<div class="bandeau">
		<a href="./" title="{{ html .Site.Other.Title }} - Cliquez ici pour retourner &agrave; la page d'accueil.">La b&eacute;tonneuse</a>
	</div>
	<div id="navigation">
		<ul>{{ range $i, $page := .Site.Pages }}
			<li><a href="{{ $page.Path }}" title="{{ if $page.Other.Description }}{{ html $page.Other.Description }}{{ else }}{{ html $page.Title }}{{ end }}">{{ html $page.Title }}</a></li>{{ end }}
		</ul>
	</div>
	<div id="contenu">
{{ end }}

{{ define "footer" }}
		<div class="validation">
			<a href="http://validator.w3.org/check?uri=referer" title="Ce site est conforme au XHTML 1.0 transitionnel."
				onclick="this.target='_blank';">
				<img src="{{ .Rel "resources/images/valid_xhtml_10.png" }}" width="40" height="14" alt="Conforme au XHTML 1.0 transitionnel." /></a>
			<a href="http://jigsaw.w3.org/css-validator/check/referer" title="Ce site est conforme &agrave; CSS 2."
				onclick="this.target='_blank';">
				<img src="{{ .Rel "resources/images/valid_css.png" }}" width="40" height="14" alt="Conforme &agrave; CSS 2." /></a>
		</div>
	</div>
</body>
</html>
{{ end }}

{{ define "page" }}{{ template "header" . }}
	{{ .Content }}
{{ template "footer" . }}{{ end }}
```

In the resulting pages, I get
```
	<div id="navigation">
		<ul>
			<li><a href="index.html" title="Page d&#39;accueil">Accueil</a></li>
			<li><a href="logiciels.html" title=""></a></li>
		</ul>
	</div>
...

	title: Logiciels
description: Mes logiciels
----
<p>Voici la liste des logiciels que je vous propose  :</p>
<p>
	<ul class="logiciels">
		<li>Logiciel 1</li>
		<li>Logiciel 2</li>
	</ul>
</p>
```
As you can see, the meta-data isn't used for the "logiciels" page. What am I doing wrong?

It also contains
```
<title>Accueil - La bétonneuse</title>
```
in the header, wile I expected
```
<title>Accueil - La b&eacute;tonneuse</title>
```

Thanks a lot!

**Comment by krpors** [comment]
Derp. Seems like this is the same thing I reported in #57. @LaBetonneuse : is it possible to test this with the latest version so this one (#26) and #57 can be closed? :D

**Comment by piranha** [comment]
Fixed in 2.4.
